.changes/next-release/feature-Configuration-22255.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
+  "category": "Configuration",
```

Comment:
Why are there two change log entries? Let's just use 61155 and mention the name ``cli_timestamp_format``